### ISSUE TITLE
Exception on start-up with lndmanage right after first channel

### ISSUE THREAD

**Issue by Pythoniasm** [bitromortac/lndmanage#68]
Regarding #67 I opened again a channel which is now pending. Since I do not have an active channel yet, there is another exception following the one of #67: 

```bash
Running in interactive mode. You can type 'help' or 'exit'.
Initializing node interface.
Initializing network graph.
Reading graph from file.
Traceback (most recent call last):
  File "/home/admin/scripts/lndmanage/py37venv/bin/lndmanage", line 8, in <module>
    sys.exit(main())
  File "/home/admin/scripts/lndmanage/py37venv/lib/python3.7/site-packages/lndmanage/lndmanage.py", line 425, in main
    node = LndNode(config_file=config_file)
  File "/home/admin/scripts/lndmanage/py37venv/lib/python3.7/site-packages/lndmanage/lib/node.py", line 83, in __init__
    self.set_info()
  File "/home/admin/scripts/lndmanage/py37venv/lib/python3.7/site-packages/lndmanage/lib/node.py", line 295, in set_info
    active_only=False, public_only=False)
  File "/home/admin/scripts/lndmanage/py37venv/lib/python3.7/site-packages/lndmanage/lib/node.py", line 338, in get_open_channels
    c.total_satoshis_received) / (age_days / 7))
ZeroDivisionError: float division by zero
```

This is most likely caused by ``age_days`` being ``0`` after node migration with new channels.

An exception handling would be useful.

**Comment by bitromortac** [comment]
Thanks for the exception reports! Will get to them in a few days!

**Comment by bitromortac** [comment]
Ok took more than a few days :smile:.

Fixed by be145a3f8600cf92b9031b215239f9112d82a8cf.
